### PR TITLE
[zh] Localize demo/_index

### PR DESCRIPTION
**Pull request by my-git9**

### CODE REVIEW

File: content/zh/docs/demo/_index.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+| Ruby       |                                                    | [Email Service](services/email/)                                                                                                         | [Email Service](services/email/)                                                             |
+| Rust       |                                                    | [Shipping Service](services/shipping/)                                                                                                   | [Shipping Service](services/shipping/)                                                       |
+-->
+| 语音        | 自动插桩                                            | 仪器化库                                                                                                                                    | 手动插桩                                                                       |
+| ---------- | -------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------- |
```
Comment (windsonsea) [inline content/zh/docs/demo/_index.md:66]:
```suggestion
| 语音 | 自动插桩 | 仪器化库  | 手动插桩  |
| ---- | -------- | ------- | -------- |
```
We can remove the redundant spaces to make it prettier.
Same issues for the following rows.

File: content/zh/docs/demo/_index.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+- [图片提供商服务](services/imageprovider/)
+
+<!--
+## Scenarios
+
+How can you solve problems with OpenTelemetry? These scenarios walk you through
+some pre-configured problems and show you how to interpret OpenTelemetry data to
+solve them.
+
+We'll be adding more scenarios over time.
```
Comment (windsonsea) [inline content/zh/docs/demo/_index.md:122]: The translation for these text is missing.

File: content/zh/docs/demo/_index.md (hunk 1 above)
Comment (reyang) [reply content/zh/docs/demo/_index.md:66]: "仪器化库" -> "插桩库"? https://github.com/open-telemetry/opentelemetry.io/blob/main/content/zh/docs/concepts/components.md#%E6%8F%92%E6%A1%A9%E5%BA%93-instrumentation-libraries

File: content/zh/docs/demo/_index.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+- [购物车服务](services/cart/)
+- [结账服务](services/checkout/)
+- [电子邮箱服务](services/email/)
+- [前端](services/frontend/)
+- [负载生成器](services/load-generator/)
+- [支付服务](services/payment/)
+- [产品目录服务](services/product-catalog/)
+- [报价服务](services/quote/)
+- [推荐服务](services/recommendation/)
+- [运输服务](services/shipping/)
```
Comment (reyang) [inline content/zh/docs/demo/_index.md:112]: 运输 -> 发货

**Review by windsonsea — APPROVED** [review]

**Pull request merged**